app/models/irregular_income.rb

```diff
@@ -4,14 +4,6 @@ class IrregularIncome < ApplicationRecord
   validates :income_type, inclusion: %w[student_loan]
   validates :frequency, inclusion: %w[annual]
 
-  def as_json(_options = nil)
```

Comment:
Is the plan to remove this whole model in a future PR?

Reply:
Yeah, I was going to do a tidy up PR. I just removed this block now to get the coverage to pass